pkg/jobs/delegate_control_test.go

```diff
@@ -350,6 +350,17 @@ func TestJobControlByType(t *testing.T) {
 	th, cleanup := newTestHelperForTables(t, jobstest.UseSystemTables, argsFn)
 	defer cleanup()
 
+	// Disable automatic statistics collection so that it does interrupt with the
```

Comment:
Do you mean "so it does not interfere"?

Also i don't understand how this causes the issue. I thought the test filters on the job type? What was the interference?